File: VineyardSite/VineyardSite/Model/OrderItem.cs
using System.Text.Json.Serialization;

namespace VineyardSite.Model;

public class OrderItem
{
    public Guid Id { get; set; }
    public WineVariant WineVariant { get; set; }
    public int WineVariantId { get; set; }
    public int Quantity { get; set; }
    [JsonIgnore]
    public Order Order { get; set; }
    public Guid OrderId { get; set; }
    
}

File: VineyardSite/AngularFrontend/VineyardSiteAngular/src/app/Models/OrderItem.ts
import { IOrder } from "./Order";
import { IWineVariant } from "./WineVariant";

export interface IOrderItem {
    Id: string,
    WineVariant: IWineVariant,
    WineVariantId: number,
    Quantity: number,
    Order: IOrder,
    OrderId: string
}
